README.md

```diff
@@ -15,39 +15,78 @@ Feel free to make the design your own.  Here are some real calculator UIs that y
 <img src="./images/exampleScreenshotTwo.png" alt="Example Two" width="400">
 <img src="./images/exampleScreenshotThree.png" alt="Example Three" width="400">
 
-## Functionality
+## User Stories, Acceptance Criteria, and Code Quality Rubric
 
-You must implement all features in the MVP list. After those are completed, look at and implement as many of the stretch features as you want to.
+You must implement all of the user stories and acceptance criteria below. After those are completed, look at and implement as many of the stretch features as you want to.
 
-### MVP
+### User Stories
 
-Ensure that your app has each of the following:
+1. I can see a calculator UI.
+1. I can enter readable positive and negative integers.
```

Comment:
I might change the wording to 'I can enter and see positive and negative integers.'